index.html

```diff
@@ -194,6 +194,38 @@ <h1>Skills</h1>
       </li>
     </ul>
   </section>
-
+  <section class="form_section">
+    <h2 id="form_title">I'm always interested in hearing about new projects,
+      so if you'd like to chat please get in touch.</h2>
+    <form action="https://formspree.io/f/xzbyoodn" method="POST" id="mobile">
+      <ul> 
+        <li>
+          <input type="text" id="full_name" name="full_name" maxlength="30" placeholder="Full name" required>
+        </li>
+        <li>
+          <input type="email" id="email" name="user_email" maxlength="30" placeholder="Email address" required>
+        </li>
+        <li>
+          <textarea id="message" name="user_message" maxlength="500" placeholder="Enter text here"></textarea>
+        </li>
```

Comment:
- [ ] Kindly add the `required` attribute on the `textarea` field to stop the user from sending an empty message for that field. This attribute is there to activate the validation upon submission of the form. For this project, you were required to have it set. Please fix that. 👍